test/test_cpp_extensions_jit.py

```diff
@@ -118,6 +120,10 @@ def test_jit_cuda_extension(self):
 
         # 2 * sigmoid(0) = 2 * 0.5 = 1
         self.assertEqual(z, torch.ones_like(z))
+        if TEST_ROCM:
+            os.unlink('cpp_extensions/hip/hip_extension.cpp')
+            os.unlink('cpp_extensions/hip/hip_extension.hip')
+            os.rmdir('cpp_extensions/hip/')
```

Comment:
Hmm, this seems very error prone (especially since people may add new tests without thinking about rocm, and they wold not be aware they have to delete the temporary files. So unfortunately I think it is probably better for the code to handle cleaning up temporaries itself.

Reply:
They will be reminded by the CI, though.

Reply:
Yes, but only in ROCm CI; which they're at much higher risk fo ignoring (sorry, wish it wasn't true, but that's the reality)